requirements.txt

```diff
@@ -0,0 +1,53 @@
+altgraph==0.10.2
```

Comment:
There are a lot of transitive dependencies in this file. You need only top level dependencies. To get them, please install `pip install pipdeptree` and run `pipdeptree`. You will see the output like 
```bash
pipdeptree==0.10.1 # skip this and its tree as pipdeptree is a debug tool 
  - pip [required: >=6.0.0, installed: 7.1.2] 
pytest==3.2.0 # add to requirements.txt
  - py [required: >=1.4.33, installed: 1.4.34] # transitive = skip
  - setuptools [required: Any, installed: 18.2] # transitive = skip
selenium==3.4.3 # add to requirements.txt
```

@qaivanpleten, please leave only main dependencies.